questions.md

```diff
@@ -0,0 +1,3 @@
+# Questions
+
+* Why is it call a `pull request`, this should be a `merge request`, right?
```

Comment:
Interesting question! Basically, it comes from early git terminology where we didn't have these collaborative tools, you had to request that the maintainer _pull_ your code, review it, and merge it with their main project. Therefore, GitHub used this terminology to describe asking a maintainer to pull your changes and review them, and then merge.